docs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+# Setup:
+#
+#     # 1. install OTP 24+
+#     # 2. install ExDoc:
+#     $ mix escript.install github elixir-lang/ex_doc
+
+rebar3 compile
+rebar3 as docs edoc
```

Comment:
```suggestion
rebar3 edoc
```

This shouldn't be needed, the `edoc` command automatically runs in the `docs` profile.

Reply:
Thanks, didn't know that 😅